app/models/iiif_manifest_presenter.rb

```diff
@@ -100,6 +100,7 @@ def metadata_field(label, value, default = nil)
   end
 
   def combined_display_date(media_object)
+    #FIXME Does this need to change now that date_issued is not required and thus could be nil
     result = media_object.date_issued
     result += " (Creation date: #{media_object.date_created})" if media_object.date_created.present?
```

Comment:
I think this will need to be changed. If someone provides a creation date but not an issued date then this method will raise an exception as written.

Reply:
@joncameron What do you think about un-combining the display date so it is two separate metadata values in the IIIF manifest?
```
"Publication date: #{media_object.date_issued}" if media_object.date_issued.present?
"Creation date: #{media_object.date_created}" if media_object.date_created.present?
```

Reply:
I would love it—that makes a lot more sense and especially with this change. I created #6050 for changing the manifest generation to separate the date fields.

Reply:
Since a new issue has been created, I'll go ahead and approve this PR :+1: